.ci/changelog.tmpl

```diff
@@ -18,6 +19,8 @@
 		{{$improvements = append $improvements (renderReleaseNote .) -}}
 	{{ else if eq "bug" .Type -}}
 		{{$bugs = append $bugs (renderReleaseNote .) -}}
+	{{ else if eq "no-release-note" .Type -}}
+		{{$nonote = append $nonote (renderReleaseNote .) -}}
```

Comment:
actually, this will output it in the release note so we don't want this

Reply:
I don't think so, since there's no `range $nonote` that outputs anything this will just build a list that doesn't get consumed. My local testing seemed to confirm this.

Reply:
Personal preference: just don't append them to the list / create the list at all, and have the else if block be empty.